src/main/java/com/jmb/springfactory/model/entity/Task.java

```diff
@@ -42,10 +39,15 @@
 
   private String reasonRejection;
 
+  private String blockedReason;
+
   private Integer orderNumber;
 
   // TODO: cambiar por one to one
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)